inc/sairoute.h

```diff
@@ -142,35 +144,42 @@ typedef sai_status_t (*sai_create_route_fn)(
  *
  * Note: IP prefix/mask expected in Network Byte Order.
  *
+ * @param[in] switch_id - Switch id
  * @param[in] route_entry - route entry
  *
  * @return #SAI_STATUS_SUCCESS on success Failure status code on error
  */
 typedef sai_status_t (*sai_remove_route_fn)(
-        _In_ const sai_route_entry_t *route_entry);
+        _In_ sai_switch_id_t switch_id,
+        _In_ const sai_route_entry_t *route_entry, 
```

Comment:
@kcudnik , the switch_create api is different than all other create_api as the switch_id is an out parameter.